aws/resource_aws_codeartifact_repository.go

```diff
@@ -61,12 +62,12 @@ func resourceAwsCodeArtifactRepository() *schema.Resource {
 			},
 			"external_connections": {
 				Type:     schema.TypeList,
-				Computed: true,
+				Optional: true,
```

Comment:
Is a change from `Computed` to `Optional` a breaking change?
If users had configured external connections outside Terraform they will now be removed unless configured in the Terraform code.
I think a change to `Computed` and `Optional` will solve this.